uint8 NUM_MOTOR_OUTPUTS = 8

uint32 motor_number				# number of motor to spin
float32 value					# output power, range [0..1]
